tests/core/editable/manual/enterbr.html

```diff
@@ -0,0 +1,12 @@
+<h3>Copy this:</h3>
+<p>a</p>
+<p>b</p>
+
```

Comment:
I would add a break line here, because sometimes when you take the selection two wide (so you end below `b` letter) it will also copy some labels from the editor. With break line, it is not happening.
```suggestion
<br />
```